src/generators/image_generators/stable_diffusion_with_loras.py

```diff
@@ -178,8 +174,7 @@ def request_camp_image_generation(self, context: AgentContext) -> Task:
             prompt=server_settings.camp_image_prompt,
             negative_prompt=server_settings.camp_image_negative_prompt,
             template_vars={
-                "genre": game_state.genre or "Adventure",
```

Comment:
question: asked above -- is `genre` no longer important for this setting?

Reply:
Genre was discussed to be somewhat duplicative to the combination of "tone" and "background", and thus was removed for now.

Reply:
It looks like it was collapsed into "narrative_tone" and "narrative_voice"?